### PR TITLE
fix type hints and doc comments

### PR DESCRIPTION
**Pull request by Ma27**

### CODE REVIEW

File: Test/WebTestCase.php [hunk 1]
```diff
@@ -554,7 +555,7 @@ protected function getUrl($route, $params = array(), $absolute = false)
      *
      * @return void
      */
-    public function isSuccessful($response, $success = true, $type = 'text/html')
+    public function isSuccessful(Response $response, $success = true, $type = 'text/html')
```
Comment (lsmith77) [inline Test/WebTestCase.php]: since others might have extended this method it would be a BC break. then again most users likely didn't extend this method. still imho as we are looking towards a 2.0 release, this change should be in a separate PR that we will merge once we begin on 2.0

Comment (Ma27) [inline Test/WebTestCase.php]: well, I don't know the history of that bundle (which I do really love by the way), but the ApiDoc tells that a response object is requried and this method uses internally the methods of the symfony response object so I cannot understand why there's a BC break.

**Comment by Ma27** [comment]
ok, so I should revert this change and open another PR for this change @lsmith77 ??

**Comment by lsmith77** [comment]
@Ma27 yes

**Comment by lsmith77** [comment]
sorry for not having gotten back earlier :-/

**Comment by lsmith77** [comment]
master was cleaned up quite a bit now via styleci
